pkg/apis/k0s.k0sproject.io/v1beta1/kuberouter.go

```diff
@@ -21,6 +21,8 @@ type KubeRouter struct {
 	AutoMTU bool `json:"autoMTU"`
 	// Override MTU setting (autoMTU must be set to false)
 	MTU int `json:"mtu"`
+	// Activate Hairpin Mode (allow a Pod behind a Service to communicate to its own ClusterIP:Port)
+	HairPinMode bool `json:"hairpinMode"`
```

Comment:
I think the P should better be lowercase, as kube-router also uses that spelling.

```suggestion
	HairpinMode bool `json:"hairpinMode"`
```

Reply:
Done in 6bffd9db5e92ca4cec2c98679dcee9ebfbacdc47 - i'll do a squash when everything is ready.